nixos/modules/services/web-servers/authelia.nix

```diff
@@ -0,0 +1,397 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+
+let
+  cfg = config.services.authelia;
+
+  format = pkgs.formats.yaml { };
+  configFile = format.generate "config.yml" cfg.settings;
+
+  autheliaOpts = with lib; { name, ... }: {
+    options = {
+      enable = mkEnableOption (lib.mdDoc "Authelia instance");
```

Comment:
> with lib
> lib.mdDoc

Another day, another inexplicable idiosyncrasy.

Reply:
And I guess you've never missed one line 🙃